### ISSUE TITLE
Using preprocessor as inpaint_global_harmonius

### ISSUE THREAD

**Issue by ajkrish95** [huggingface/diffusers#4978]
### Model/Pipeline/Scheduler description

**Model/Pipeline/Scheduler description**

Automatic1111 supports preprocessor of inpaint_global_harmonius for the Inpaint ControlNet model over web ui. Is this functionality already present in the DIffusers library and if not can it be added?

**Provide useful links for the implementation**
https://github.com/Mikubill/sd-webui-controlnet/discussions/1143

### Open source status

- [X] The model implementation is available
- [ ] The model weights are available (Only relevant if addition is not a scheduler).

**Comment by patrickvonplaten** [comment]
I don't think this is something we necessarily are planning to have in "core" diffusers - could we maybe just try to build a code snippet that showcases this feature and put it in a github gist or community pipeline?
